src/app/components/tv-show-table/tv-show-table.component.ts

```diff
@@ -9,5 +10,6 @@ import { CommonModule } from '@angular/common';
   styleUrls: ['./tv-show-table.component.css']
 })
 export class TvShowTableComponent {
-
+  @Input() tvShows: ITvShow[] = [];
+  @Input() fetchComplete: boolean = true;
```

Comment:
Angular 17 tip: Consider using the [new input() functions](https://www.angulartraining.com/daily-newsletter/whats-new-in-angular-17-1/)

Reply:
Very cool